src/hooks/useQuota/useQuota.test.tsx

```diff
@@ -442,6 +536,50 @@ describe("useQuota", () => {
     });
   });
 
+  describe("quota should check only non-appeal products", () => {
+    it("should set quota state to DEFAULT if non-appeal products quantities are non-zero", async () => {
+      expect.assertions(1);
+
+      let ids = ["ID1"];
+      mockGetQuota.mockResolvedValueOnce(
+        mockQuotaResSingleIdWithAppealProducts
+      );
+      const { result, waitForNextUpdate } = renderHook(
+        () => useQuota(ids, key, endpoint),
+        {
+          wrapper,
+          initialProps: {
+            products: defaultAppealProducts
+          }
+        }
+      );
+      await waitForNextUpdate();
+
+      expect(result.current.quotaState).toStrictEqual("DEFAULT");
+    });
+
+    it("should set quota state to NO_QUOTA if non-appeal products quantities are zero", async () => {
```

Comment:
Should this be `if appeal products quantities are zero`?

Reply:
In this test, I am checking that the quota state is determined only by non-appeal products.
Let me know if the test name or variable names in the tests are confusing!